modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.tsx

```diff
@@ -61,14 +62,16 @@ const Card = ({item, schema}: {item: any; schema: ICardSchema}) => {
 	const localizedTitle = getLocalizedValue(item, schema.title)?.value || '';
 	const selectedItemKey = selectedItemsKey && item[selectedItemsKey];
 	const formattedActions =
-		actionsRef.current &&
-		(filterItemActions(actionsRef.current, item) as any);
+		actionsRef?.current &&
+		(filterItemActions(actionsRef?.current, item) as any);
```

Comment:
The optional chaining here is odd. Won't `actionsRef` always exist? The value of `actionsRef.current` might be undefined, existing code checks that.

Reply:
It was an editor suggestion. Just change to remove warning, but I'll revert it back.
![image](https://github.com/user-attachments/assets/96d31f12-f96b-4fcb-9be8-e443293c51d2)